### PR TITLE
Add net8 target framework

### PR DESCRIPTION
**Pull request by stiankroknes**
Add net8 target framework and conditionally select version for Microsoft.Extensions.FileSystemGlobbing based on framework.

### CODE REVIEW

**Comment by stefanloerwald** [comment]
Hi @stiankroknes,

As net8.0 still relies on preview packages, can you please add a note about that in the Readme?

Thanks
Stefan

**Comment by stiankroknes** [comment]
Hi, not sure about next version, but something like this: 

#### Changes in version 3.6.X

Added support for .NET 8. Note that this release depends on preview version of `Microsoft.Extensions.FileSystemGlobbing` package.

Ok?

-Stian

File: WebCompiler/WebCompiler.csproj [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+		<PackageReference Include="NUglify" Version="1.20.7" />
+	</ItemGroup>
+	<ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+		<PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="6.0.0" />
+	</ItemGroup>
+	<ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
+		<PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="7.0.0" />
+	</ItemGroup>
+	<ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
+		<PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="8.0.0-rc.*" />
```
Comment (MaxymGorn) [inline WebCompiler/WebCompiler.csproj:32]:
```suggestion
		<PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="8.0.0" />
```

**Comment by stefanloerwald** [comment]
net8.0 is now officially supported. Closing this PR.

**Pull request closed without merging**